src/main/java/school/lemon/changerequest/java/extendedinteger/ExtendedInteger.java

```diff
@@ -1,119 +1,163 @@
 package school.lemon.changerequest.java.extendedinteger;
 
+
 /**
  * Integer representation with some additional features
  */
 public class ExtendedInteger {
+    private int value;
 
     public ExtendedInteger(int value) {
-        //TODO: implement me
+        this.value = value;
     }
 
     /**
      * Check whether specified value is even
+     *
      * @param value to check
      * @return true if value is even, false - otherwise
      */
     public static boolean isEven(int value) {
-        //TODO: implement me
+        if (value % 2 == 0) {
+            return true;
+        }
         return false;
     }
 
     /**
      * Check whether specified value is odd
+     *
      * @param value to check
      * @return true if value is odd, false - otherwise
      */
     public static boolean isOdd(int value) {
-        //TODO: implement me
+        if (value % 2 != 0) {
+            return true;
+        }
         return false;
     }
 
     /**
      * Check whether specified value is prime
+     *
      * @param value to check
      * @return true if value is prime, false - otherwise
      */
     public static boolean isPrime(int value) {
-        //TODO: implement me
+        int i;
+        if (value > 1) {
+            for (i = 2; i <= value / 2; i++) {
+                if (value % i == 0) {
+                    return false;
+                }
+            }
+            return true;
+        }
         return false;
     }
 
     /**
      * Parse specified char array and create instance of {@code ExtendedInteger}
+     *
      * @param value to parse
      * @return instance of {@code ExtendedInteger} or
      * null in case specified value is null or the value does not contain a parsable integer
      */
     public static ExtendedInteger parseInt(char[] value) {
-        //TODO: implement me
+        if (value.length != 0 && value != null) {
```

Comment:
1. Looks like you'll have NPE on null. To make it safe you should check for null before check for size. 
2. I'd prefer to invert this condition, to decrease code in brackets like:
```
if (value == null || value.length == 0) 
    return null;
```